src/Telegram.Bot/Telegram.Bot.csproj

```diff
@@ -3,22 +3,22 @@
     <TargetFramework>netstandard1.1</TargetFramework>
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
     <Product>Telegram Bot API</Product>
-    <Title>Telegram Bot API</Title>
-    <Description>Telegram Bot Library supporting Bot Api 3.3, Payments, Inline, Group &amp; Channel management</Description>
-    <Authors>Robin Müller</Authors>
+    <Title>Telegram Bot API: .NET Client</Title>
```

Comment:
I would Change this to `Telegram Bot API Client` as it's obvious that the library is for .NET. What do you think